### ISSUE TITLE
Help in running cfSort with pre-trained models

### ISSUE THREAD

**Issue by ccarlo21** [jasminezhoulab/cfSort#2]
Hi, a very interesting work!
I’d like to use cfSort with some WGBS samples using the pre-trained DNN model you provided.
Can I ask for the scripts that transform the mapped reads or the methylation signals obtained from Bismark into the format needed for the test_filename? I am not 100% sure on what should I do.
Can you also provide the norm params used in the paper? I see that I could obtain norm_params from the demo train files, but they are just a small fraction of all the samples you used for the training of the provided DNN model.
Thank you in advance

**Comment by DashuoLI** [comment]
Hi Carlo, Thanks for your interest! 

We uploaded the normalization parameters used in the paper to the "codes" directory. You can find them there as norm_params.npy. We also added a script, data_prep.for_test_only.py, to convert the testing data matrix in a tab-delimited format to the input data for the DNN prediction.

We described the detailed feature construction in our manuscript. We are organizing the data preprocessing pipeline. It will be ready to release by early August.
